### ISSUE TITLE
Gagging sound playing too often

### ISSUE THREAD

**Issue by kevinfoley** [eezstreet/SWATEliteForce#188]
The "gagging" sound effects that can play for an incapacitated suspect seem too common on some levels. For example, in Stetchkov warehouse, every single suspect you incapacitate will make those sound effects. I'm not sure if this is a bug in the code, or just bad level design (e.g. if the same voice was applied to every single enemy in the level) but it's very annoying.

**Comment by eezstreet** [comment]
I think it's more of a level design issue with those suspects specifically. The Stetchkov types are just excessively annoying. The volume for those guys can probably be turned down.

**Comment by kevinfoley** [comment]
@eezstreet Is it possible and within the scope of the mod to edit the level and maybe assign more voices?

**Comment by eezstreet** [comment]
Yeah, it should be. It's only really SoundEffects.ini (?) that needs to be changed. The volume of their sound effects just needs to be reduced.

________________________________

**Comment by kevinfoley** [comment]
I strongly feel that changing out some of the voices is a better solution than reducing the volume of the sound effect. The gagging is too loud, but the bigger issue is that literally ever single suspect in the entire level plays the exact same sound effect if they are incapacitated.

**Comment by eezstreet** [comment]
Yeah, I wonder if that can be fixed somehow, maybe with new voice acting? I'm not entirely sure.

**Comment by theengineertcr** [comment]
There are 4 different downs for male bulgarian 1 & 2. We can make bulgarian 1 use the second voice as it has less gagging.
